docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md

```diff
@@ -7,6 +7,13 @@ in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## UNRELEASED
```

Comment:
Marking this as unreleased first as intend to do the version bump along with other sidebar APIs being introduced.